src/SDKs/Resource/Net45.Tests/Net45.Tests.csproj

```diff
@@ -0,0 +1,71 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />-->
+  <PropertyGroup>
+    <TargetFrameworks>net452</TargetFrameworks>    
+    <PackageId>Resource.Net452.Tests</PackageId>
+    <Description>Resource.Net452.Tests Class Library</Description>
+    <AssemblyName>Net45.Tests</AssemblyName>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
```

Comment:
@mjfrb is there a reason you had to create the test project this way.
can you model your test project looking at other test projects (e.g. billing)
especially please let me know why you need the below properties
- all properties related to nuget package
- Signing
- you do not need any package reference related to xunit, they are implicitly added to all test projects